js/lesson-24/part5.js

```diff
@@ -0,0 +1,28 @@
+`use strict`;
+
+let $number;
+let $pos = 0;
+let $neg = 0;
+let $zero = 0;
+let $even = 0;
+let $odd = 0;
+
+for (i = 1; i <= 10; i++) {
+    $number = prompt(`Введіть число`);
+    if ($number === `` || isNaN($number)) {
+        console.log(`Не прийнято! Спробуйте ще раз спочатку.`);
+        break;
+    } else {
+        if (Number($number) == 0) {
+            $zero += 1;
+        } else {
+            Number($number) > 0 ? $pos += 1 : $pos = $pos;
+            Number($number) < 0 ? $neg += 1 : $neg = $neg;
```

Comment:
You don't need to `$neg = $neg`. Just add `: null` or `: undefined`, since we care only about `Number($number) < 0`

Reply:
Fixed